### ISSUE TITLE
why i cannot create multiple requets? 

### ISSUE THREAD

**Issue by Akin1995** [gradio-app/gradio#7562]
### Describe the bug

doing multiple requests creates error and then work again after some time, what can i do and why is it so?

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

gradio_interface = gui.gradio_interface.queue(concurrency_count=50, max_size=50)
# gradio_interface.queue(default_concurrency_limit=50, max_size=30)

app = gradio.mount_gradio_app(app, gradio_interface, path="/gradio")
if __name__ == "__main__":  # for debugging locally
    uvicorn.run("main:app", host="127.0.0.1", port=8000, workers=1)

### Logs

```shell
INFO:     connection open
INFO:     connection closed
INFO:     connection closed
INFO:     ('127.0.0.1', 49152) - "WebSocket /gradio/queue/join" [accepted]
INFO:     connection open
INFO:     ('127.0.0.1', 49162) - "WebSocket /gradio/queue/join" [accepted]
INFO:     connection open
INFO:     connection closed
INFO:     connection closed
INFO:     ('127.0.0.1', 42068) - "WebSocket /gradio/queue/join" [accepted]
INFO:     connection open
INFO:     ('127.0.0.1', 36750) - "WebSocket /gradio/queue/join" [accepted]
INFO:     connection open
INFO:     ('127.0.0.1', 41186) - "WebSocket /gradio/queue/join" [accepted]
INFO:     connection open
INFO:     connection closed
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/uvicorn/protocols/websockets/websockets_impl.py", line 250, in run_asgi
    result = await self.app(self.scope, self.asgi_receive, self.asgi_send)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/applications.py", line 123, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/middleware/errors.py", line 151, in __call__
    await self.app(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/middleware/cors.py", line 75, in __call__
    await self.app(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 762, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 782, in app
    await route.handle(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 485, in handle
    await self.app(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/applications.py", line 123, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/middleware/errors.py", line 151, in __call__
    await self.app(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/middleware/cors.py", line 75, in __call__
    await self.app(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 762, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 782, in app
    await route.handle(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 373, in handle
    await self.app(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 96, in app
    await wrap_app_handling_exceptions(app, session)(scope, receive, send)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/routing.py", line 94, in app
    await func(session)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/fastapi/routing.py", line 360, in app
    await dependant.call(**values)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/gradio/routes.py", line 571, in join_queue
    session_info = await asyncio.wait_for(
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/asyncio/tasks.py", line 489, in wait_for
    return fut.result()
           ^^^^^^^^^^^^
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/websockets.py", line 133, in receive_json
    self._raise_on_disconnect(message)
  File "/home/akin/miniconda3/envs/ml_toolkit/lib/python3.11/site-packages/starlette/websockets.py", line 105, in _raise_on_disconnect
    raise WebSocketDisconnect(message["code"], message.get("reason"))
starlette.websockets.WebSocketDisconnect: (1006, None)
```

### System Info

```shell
anthropic==0.9.*
deepl==1.16.*
elevenlabs==0.2.*
google-cloud-aiplatform==1.39.*
langdetect==1.0.*
openai==1.7.*
termcolor==2.4.*
tiktoken==0.5.*

# webapp
fastapi==0.109.*
pydantic==2.5.*
pydantic-settings==2.1.*
uvicorn[standard]==0.25.*
gradio==3.47.*

# database
pymongo[srv]==4.6.*
motor==3.3.*
beanie==1.24.*

# GCP
google-cloud-core==2.4.*
google-cloud-logging==3.9.*
google-cloud-pubsub==2.19.*

# test requirements
coverage==7.4.*
pytest==7.4.*
pytest-asyncio==0.23.*
httpx==0.26.*
mongomock-motor==0.0.*

# Authentication
# python-jose[cryptography]

toml==0.10.*
```

### Severity

Blocking usage of gradio

**Comment by abidlabs** [comment]
Hi @Akin1995 the code is hard to parse, but it looks like from the error that you're using Gradio 3.x, which we don't officially support. Can you try upgrading to the latest version of Gradio?

**Comment by Akin1995** [comment]
We cant because we want to keep on gradio 3.47. Which Part you cannot read ?

**Comment by abidlabs** [comment]
> gradio_interface = gui.gradio_interface.queue(concurrency_count=50, max_size=50)
> app = gradio.mount_gradio_app(app, gradio_interface, path="/gradio")
> if name == "main": # for debugging locally
> uvicorn.run("main:app", host="127.0.0.1", port=8000, workers=1)

This is not sufficient to reproduce the issue

Why do you. need to keep running Gradio 3.47? We strongly encourage upgrading to 4.x as that is the supported version that we can help debug

**Comment by Akin1995** [comment]
Could you give me an advice which code could be nessecary? I am a Junior!

**Comment by Akin1995** [comment]
> > gradio_interface = gui.gradio_interface.queue(concurrency_count=50, max_size=50)
> > app = gradio.mount_gradio_app(app, gradio_interface, path="/gradio")
> > if name == "main": # for debugging locally
> > uvicorn.run("main:app", host="127.0.0.1", port=8000, workers=1)
> 
> This is not sufficient to reproduce the issue. Can you please provide a minimal code example that we can use to reproduce the issue above? See: https://stackoverflow.com/help/minimal-reproducible-example
> 
> Also, can you clarify why do you need to keep running Gradio 3.47? We strongly encourage upgrading to 4.x as we can help support / debug that with you

i added some code

**Comment by abidlabs** [comment]
Hi @Akin1995 for a repro, can you please provide standalone code that I can run in a fresh environment that reproduces the issue you're describing? The code that is provided here cannot be run because it includes external dependencies (anything in the .src folder)

**Comment by Akin1995** [comment]
the problem is you need some access to some models gpt 4 etc

**Comment by abidlabs** [comment]
The Gradio part should just involve the UI, so you should mock these requests and use placeholder values instead. Without this, I'm sorry but we won't be able to help solve the issue.

**Comment by Akin1995** [comment]
how can i achieve this ?

**Comment by Akin1995** [comment]
i have a question what is the return type of queue() method? is there any return type?

**Comment by Akin1995** [comment]
> The Gradio part should just involve the UI, so you should mock these requests and use placeholder values instead. Without this, I'm sorry but we won't be able to help solve the issue.


i have a question what is the return type of queue() method? is there any return type?

**Comment by abidlabs** [comment]
I'll go ahead and close this for now @Akin1995 -- if we have a repro, we can reopen
